members/federal_us_epa_adm_scott_pruitt.yaml

```diff
@@ -2,6 +2,7 @@ bioguide: federal_us_epa_adm_scott_pruitt
 contact_form:
   method: post
   action: ""
+  usePhantomJsLogging: true
```

Comment:
This is fine, but no longer necessary since the `javascript` step will force the usage of phantom